art.py

```diff
@@ -0,0 +1,110 @@
+from os import listdir
+from os.path import join, splitext, dirname, isfile
```

Comment:
can this be `from os import path` so that it is clear what some functions in here are doing (operating on paths)?